detox/src/devices/DeviceRegistry.js

```diff
@@ -59,6 +67,18 @@ class DeviceRegistry {
 
     this._lockfile.write(newState);
   }
+
+  static ios() {
```

Comment:
what about... `createIos()`, `createAndroid()`? plain `ios` is a bit under-verbose